tests/spec/core/figures-spec.js

```diff
@@ -81,4 +81,28 @@ describe("Core - Figures", function() {
     expect(figLinks.item(0).textContent).toEqual("Figure 1 PREFIG");
     expect(figLinks.item(1).textContent).toEqual("Figure 2 IMGTIT");
   });
+
+  it("adds natural width and height to images if not set", async () => {
+    const imgDataURL = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAADCAIAAADUVFKvAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4gQKACEWdS72PwAAABl0RVh0Q29tbWVudABDcmVhdGVkIHdpdGggR0lNUFeBDhcAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAASUVORK5CYII=";
+    const ops = {
+      config: makeBasicConfig(),
+      body:
+        `<section>
+         <img id="image-with-id" src="${imgDataURL}">
+         <img id="image-with-dimensions" height=100 width=200 src="${imgDataURL}">
+      </section>`
+    };
+
+    const doc = await makeRSDoc(ops);
+
+    const image1 = doc.getElementById("image-with-id");
+    expect(image1).toBeTruthy();
+    expect(image1.height).toBe(3);
```

Comment:
this test is not quite correct... we want to check that `hasAttribute("height")` and `hasAttribute("height")` and check that the attribute value is as expected.